bin/test

```diff
@@ -72,9 +76,21 @@ function downloadExecutableAndRunTests() {
     getDownloadUrl(function (downloadUrl) {
         console.log('Downloading VS Code into "' + testRunFolderAbsolute + '" from: ' + downloadUrl);
 
-        var stream = remote(downloadUrl, { base: '' })
-            .pipe(vzip.src())
-            .pipe(symdest(testRunFolder));
+        var version = downloadUrl.match(/\d\.\d\.\d/)[0].split('\.');
+        var isTarGz = downloadUrl.match(/linux/) && version[0] >= 1 && version[1] >= 4;
```

Comment:
Seems very complicated, why not check for the URL ending in tar.gz and make this version independent?